bundles/ch.elexis.core.ui.contacts/META-INF/MANIFEST.MF

```diff
@@ -9,14 +9,16 @@ Bundle-Localization: plugin
 Require-Bundle: ch.elexis.core.data;bundle-version="3.10.0",
  ch.elexis.core.ui;bundle-version="3.10.0",
  ch.elexis.core.ui.medication;bundle-version="3.0.0",
- org.eclipse.ui.views.properties.tabbed;bundle-version="3.9.100"
+ org.eclipse.ui.views.properties.tabbed;bundle-version="3.9.100",
+ ch.medelexis.messaging.twilio.ui
```

Comment:
Das geht nicht. core.ui.contacts ist im elexis-3-core repository, twilio im medelexis repo, das würde alle opensource builds brechen, und auch den medelexis build. das geht nur in deiner IDE